src/finnish-fish-soup.md

```diff
@@ -0,0 +1,37 @@
+# Finnish fish soup
+
+Traditional Finnish fish soup usually served with rye bread.
+Originally from a finnish book _Saiturin Keittiö_ (The Cheapskates Kitchen) from 1992. 
+
+- ⏲️ Time: 50 min
+- 🍽️ Servings: 4
+- 💲  Cost ~8€
+
+## Ingredients
+
+- 600g of small fish (eg. perch) or fish offal (heads, fins, ...)
+- 6 midsized potatoes
+- small packet of frozen spinach or 50g of parboiled fresh spinach
+- 1 tbsp of wheat flour
+- 2 egg yellows
+- 1/2 dl of cream
+- a small lump of butter
+
+## Directions
+
+1. Boil the fishes for 30 min in 12 dl of water
```

Comment:
`dl` again. Refer to my above message

Reply:
This should be `1.2 L`. Strange way of using decilitres.